### ISSUE TITLE
mcrypt failed to build on 10.10.3

### ISSUE THREAD

**Issue by blude** [Homebrew/homebrew#39150]
Trying to install mcrypt, received the following output:

```
$ brew install mcrypt
==> Installing mcrypt dependency: mhash
==> Downloading https://downloads.sourceforge.net/project/mhash/mhash/0.9.9.9/mhash-0.9.9.9.tar.gz
Already downloaded: /Library/Caches/Homebrew/mhash-0.9.9.9.tar.gz
==> ./configure --prefix=/usr/local/Cellar/mhash/0.9.9.9
==> make install
2015-04-28 14:05:56 -0300

make install

couldn't understand kern.osversion `14.3.0'
couldn't understand kern.osversion `14.3.0'

READ THIS: https://git.io/brew-troubleshooting
```

I've searched the issues and tried a couple of things but none'd seen to work.

Logging from gcc:
https://gist.github.com/anonymous/7e7365491576e19f440f

**Comment by jacknagel** [comment]
Please `brew gist-logs mhash` rather than gcc.

**Comment by blude** [comment]
Oh, my bad, here it goes:
https://gist.github.com/anonymous/c4ac58385cb070ce02cf

**Comment by jacknagel** [comment]
Hmm, usually when a command fails with no output it signals that the tool is missing... do you have a `make` on your system? `xcrun --find make`.

**Comment by blude** [comment]
Hi, @jacknagel! Apparently, yes, got the following result: `/Applications/Xcode.app/Contents/Developer/usr/bin/make`.

Should this be on my `$PATH` variable?

**Comment by jacknagel** [comment]
Does `/usr/bin/make` exist, and does it work (e.g. run `/usr/bin/make --version`)?

**Comment by unfunco** [comment]
I ran into this same issue, `/usr/bin/make` exists, and running `which make` showed I was using `/usr/bin/make`.

I resolved the issue by [downloading Mhash](http://mhash.sourceforge.net) and running the usual:

```bash
$ ./configure
$ ./make
$ brew link --overwrite mhash
```

**Comment by blude** [comment]
Indeed `make` worked fine. The output was: `GNU Make 3.81`.

Then I tried @unfunco's take but no luck. I'm afraid I got lost somewhere in the process.

So this situation was so frustrating that I decided to completely uninstall Homebrew and it actually solved the problem. Hooray! Kinda bummer that I couldn't fix it without a nuclear option.

I really appreciate all the support and consideration. Thanks @jacknagel @unfunco!
